frontend/packages/knative-plugin/integration-tests/features/Knative/create-knative-workload.feature

```diff
@@ -88,3 +88,22 @@ Feature: Create a workload of 'knative Service' type resource
               And user clicks Create button on Add page
              Then user will be redirected to Topology page
               And user is able to see workload "nodejs-ex-git" in topology page list view
+
+
+        @regression
+        Scenario: Create a knative workload with advanced option "Scaling" from From Git card
+            Given user is at Import from git page
+             When user enters Git Repo url as "https://github.com/sclorg/dancer-ex.git"
+              And user enters name as "dancer-ex-git" in General section
+              And user selects resource type as "Knative Service"
+              And user clicks "Scaling" link in Advanced Options section
+              And user enters number of Min Pods as "1"
+              And user enters number of Max Pods as "5"
+              And user enters number of Concurrency target as "3"
+              And user enters number of Concurrency limit as "15"
+              And user enters percentage of Concurrency utilization as "70"
+              And user enters value of Autoscale window as "9 Sec"
+              And user clicks Create button on Add page
+              And user clicks on the Knative Service workload "dancer-ex-git"
+              And user clicks on name KSVC "dancer-ex-git"
+             Then user will see value of autoscaling.knative.dev/targetUtilizationPercentage and autoscaling.knative.dev/window as 70 and 9s respectively
```

Comment:
It would be nice if you check for all other annotations as you enter the values for other fields. and look like this scenario is for the advanced option `Scaling`. so better to check for all the respective annotations.

Reply:
Updated